src/Core/MailTemplate/FolderThemeScanner.php

```diff
@@ -190,6 +190,15 @@ private function checkThemeFolder($mailThemeFolder)
 
     private function getTemplatePath(SplFileInfo $fileInfo): string
     {
-        return '@MailThemes' . substr($fileInfo->getRealPath(), strlen($this->baseThemeFolder));
+        $realPath = $fileInfo->getRealPath();
+
+        // If the template comes from a module, we use the @Modules twig path instead of @MailThemes
+        if (str_contains($realPath, 'modules')) {
+            // We remove everything in the path until "modules" and we append @Modules before that. 7 is the length of the string "modules"
+            return '@Modules' . substr($realPath, strpos($realPath, 'modules')+7);
```

Comment:
```suggestion
            return '@Modules' . substr($realPath, strpos($realPath, 'modules') + strlen('modules'));
```

Reply:
This way we understand explicitly where this 7 comes from

Reply:
Also, maybe use the module path const? I feel like only looking for `modules` in the path could lead to error if for any reason a theme contains a `modules` folder that contains templates

Reply:
Nothing to add to the great suggestions of @jolelievre and @atomiix 

Is it possible to also add a unit test for this usecase?

Reply:
Done